jarbas_hive_mind/__main__.py

```diff
@@ -4,22 +4,14 @@
 def main():
     import argparse
     parser = argparse.ArgumentParser(description='Start HiveMind as a server')
-    parser.add_argument("--name", help="human readable name")
-    parser.add_argument("--access_key", help="access key")
-    parser.add_argument("--crypto_key", help="payload encryption key")
-    parser.add_argument("--port", help="HiveMind port number")
+    parser.add_argument("--port", help="HiveMind port number", type=int)
     args = parser.parse_args()
-    # Check if a user was defined
-    if args.name is not None:
-        from jarbas_hive_mind.database import ClientDatabase
-        with ClientDatabase() as db:
-            db.add_client(args.name, args.access_key, crypto_key=args.crypto_key)
     config = CONFIGURATION
     listener = get_listener()
     listener.load_config(config)
     # Replace defined values
     if args.port is not None:
-        listener.port = int(args.port)
+        listener.port = args.port
```

Comment:
keep the cast to int, if a string is passed it should still work. I dont recall details how a string got there but i know this was an issue before for some users

Reply:
Solved, there was a ```type=int``` missing in the parser.